contrib/automation_tests/core/orbit_e2e.py

```diff
@@ -78,6 +78,17 @@ def find_context_menu_item(self, text: str, raise_on_failure=True):
         return self.find_control('MenuItem', text, parent=self.suite.application.top_window(),
                                  raise_on_failure=raise_on_failure)
 
+    def find_combo_box_item(self, text: str, raise_on_failure=True):
+        """
+        Specialized version of find_control for combo box items. This is provided due to the special
+        behavior of expanded combo box lists: They are not parented underneath the current main window, but
+        are treated as a separate window.
+        """
+        # Add an extra sleep. combo box needs to show up before we search for it.
+        time.sleep(1)
```

Comment:
I'd prefer to replace this with `wait_for_condition` instead of a fixed sleep time, we are trying to avoid explicit sleeps whenever possible.

Reply:
what would you put in the the lambda in wait_for_condition?

Reply:
Basically the whole body of this, you'd want to wait until the control can be found or until `wait_for_condition` times out.

e.g.:
```
wait_for_condition(lambda: self.find_control('ListItem', text, parent=self.suite.application.top_window(), raise_on_failure=false) is not None`)
```

After this returns, you'd actually need to call `self.find_control` again like below because `wait_for_condition` doesn't return the result.

Reply:
I tried exactly this (also with is not None) yesterday and it did not work. I am going to try again and then maybe we can quickly chat about this.

Reply:
I made it work🎉 Turns out the `lambda:` is a necessary keyword here and not the name of the parameter. One day I will understand python. 
Anyways. I am closing this PR, as it will be part of the e2e test PR